server/src/main/java/com/google/speech/tools/voxetta/data/Prompt.java

```diff
@@ -22,25 +22,48 @@
 public class Prompt {
 
     private final long id;
-    //    private final VendorName;
-//    private final ProjectId;
-    private final String type;
+    private final Type type;
     private final String body;
 
-    public Prompt(long id, String type, String body) {
+    enum Type {
+        TEXT,
+        IMAGE
+    }
+
+    /**
+     * @param id   {long} database ID of the prompt. param type {String} type of prompt.
+     * @param body {String} body (or content) of the prompt.
+     */
+    public Prompt(long id, String type, String body) throws IllegalArgumentException {
```

Comment:
Pass in `Type type` instead of a string, so you can directly assign it to `this.type`